R/component.R

```diff
@@ -23,7 +23,7 @@
 #'      (excluding aliased coefficients).
 #' - `beta_vcov_complete`: estimated variance-covariance matrix including
 #'      aliased coefficients with entries set to `NA`.
-#' - `varcor`: estimated covariance matrix for residuals.
+#' - `varcor`: estimated covariance matrix for residuals (by groups).
```

Comment:
better explain the structure, e.g. would it be a list of matrices if it is by groups and otherwise just a matrix?